.kokoro/presubmit/downstream-build.sh

```diff
@@ -62,8 +62,8 @@ tar -xf showcase-*
 popd
 
 # Run showcase tests with `native` profile
-pushd sdk-platform-java/showcase
+pushd sdk-platform-java/java-showcase
 mvn test -Pnative,-showcase -Denforcer.skip=true -ntp -B
```

Comment:
This "-showcase" seems a typo (before this pull request).

Reply:
Updated.